api/cpp/Cargo.toml

```diff
@@ -34,6 +34,8 @@ renderer-winit-femtovg = ["i-slint-backend-selector/renderer-winit-femtovg"]
 renderer-winit-skia = ["i-slint-backend-selector/renderer-winit-skia"]
 renderer-winit-skia-opengl = ["i-slint-backend-selector/renderer-winit-skia-opengl"]
 
+experimental = []
```

Comment:
This is a good example of the missing "what" - we'd probably not call this `slint/experimental` in the public API but rather `slint/experimental-api` or maybe `slint/experimental-features` - a "what" basically.